tests/ranged_balance.cpp

```diff
@@ -334,8 +337,18 @@ static void range_test( const std::array<double, 5> &test_thresholds )
             WARN( "No matching dispersion found" );
         } else {
             WARN( "Range: " << r << " Dispersion: " << found_dispersion );
+            data.push_back( found_dispersion );
         }
     }
+    if( write_data )
+    {
+        write_to_file( "./data/json/hit_range.json", [&]( std::ostream & fsa ){
```

Comment:
This writes a ".json" file into the data/json directory. The JSON loader (init.cpp) loads *all* ".json" files from that directory. And it expects those files to have objects with a "type" member.

This code does not write a "type" member, so the loader does not know how to load it.

I suggest adding a "type" like this `j_out.member("type", "hit_range");`

Reply:
Added in the latest commit.